redbot/cogs/customcom/customcom.py

```diff
@@ -338,24 +339,42 @@ def __init__(self, bot):
             return
 
         results = []
-
-        for command, body in response.items():
+        for command, body in cc_dict.items():
             responses = body["response"]
             if isinstance(responses, list):
                 result = ", ".join(responses)
             elif isinstance(responses, str):
                 result = responses
             else:
                 continue
-            results.append("{command:<15} : {result}".format(command=command, result=result))
-
-        _commands = "\n".join(results)
-
-        if len(_commands) < 1500:
-            await ctx.send(box(_commands))
+            # Don't put a line-break in preview
+            newline_pos = result.find("\n")
+            if newline_pos != -1:
+                result = result[:newline_pos] + "..."
+            # Cut preview to 52 characters max
+            if len(result) > 52:
+                result = result[:49] + "..."
```

Comment:
I don't like this pagification method with embed. I think this is a good case for field based pagination to allow new lines in previews. While this method is certainly cleaner looking than the previous one, It's also less functional for many purposes.

As an example, I have many CCs in a codiing discord utilizing code blocks. The only thing I get is the language specifier and escaped backticks for those.